tests/test_rio_info.py

```diff
@@ -461,4 +461,6 @@ def test_info_aws_unsigned():
     """Unsigned access to public dataset works (see #1637)"""
     runner = CliRunner()
     result = runner.invoke(main_group, ['--aws-no-sign-requests', 'info', 's3://landsat-pds/L8/139/045/LC81390452014295LGN00/LC81390452014295LGN00_B1.TIF'])
+    print(result.output)
+    print(result.stderr)
```

Comment:
@snowman2 we can remove these two lines. I know what's up with the build failures and will merge this PR despite them.

Reply:
Sounds good. Thanks!